README.md

```diff
@@ -32,12 +31,12 @@ pysatMissions uses common Python modules, as well as modules developed by
 and for the Space Physics community.  This module officially supports
 Python 3.8+.  
 
-| Common modules | Community modules |
-| -------------- | ----------------- |
-| numpy          | aacgmv2           |
-| pandas         | apexpy            |
-| pyEphem        | OMMBV>=1.0        |
-| sgp4>=2.7      | pysat>=3.0        |
+| Common modules | Community modules | Optional modules |
+| -------------- | ----------------- | ---------------- |
+| numpy          | pysat>=3.0.4      | aacgmv2          |
+| pandas         |                   | apexpy           |
+| pyEphem        |                   | OMMBV            |
+| sgp4>=2.7      |                   |                  |
```

Comment:
Are pyEphem and sgp4 also community modules?

Reply:
I could go either way on this.  I was thinking that these two were more widely used outside of our field (similar to netCDF), so I went with common.

Reply:
I was thinking more in the funding levels, not sure how well supported the maintenance is?

Reply:
Probably a good idea to move these.  pyEphem is stale, and will be dropped in 0.4.0.